components/sup/src/manager/service/config.rs

```diff
@@ -42,6 +42,7 @@ static ENV_VAR_PREFIX: &'static str = "HAB";
 /// is deeper than this value crosses into overly complex territory when describing configuration
 /// for a single service.
 static TOML_MAX_MERGE_DEPTH: u16 = 30;
+pub const CONFIG_PERMISSIONS: u32 = 0o740;
```

Comment:
Good call!

I'm assuming this permission is on a directory entry? Is there a reason to avoid the execute bit for the group (i.e. `750` vs `740`)?

Reply:
Never mind, I opened up more context and saw that this is working on a file not directory, disregard last 😄